egs/swbd/s5c/local/chain/run_xlstm.sh

```diff
@@ -0,0 +1,233 @@
+#!/bin/bash
+
+# 6i is based on run_lstm_6h.sh, but changing the HMM context from triphone to left biphone.
```

Comment:
is this comment out of date?  Perhaps give it a different letter and run the comparison script so you can update the comment?